spi_nand_flash/priv_include/nand.h

```diff
@@ -53,6 +53,7 @@ typedef struct {
     uint32_t erase_block_delay_us;
     uint32_t program_page_delay_us;
     ecc_data_t ecc_data;
+    uint8_t is_quad_enable_bit;
```

Comment:
```suggestion
    uint8_t has_quad_enable_bit;    // 1 if the chip supports enabling QIO/QOUT mode using bit 0 of 0xB0 (REG_CONFIG), 0 otherwise
```
Just a suggestion to make it easier to understand what this field means.